src/lib/routes/feedback-route.ts

```diff
@@ -0,0 +1,134 @@
+import { FastifyInstance } from "fastify";
+import supabase from "../supabase.js";
+import { UserError } from "../errors.js";
+
+const feedbackRoute = function (
+	app: FastifyInstance,
+	options: unknown,
+	next: (err?: Error | undefined) => void,
+) {
+	// app.get("/", (req, res) => {
```

Comment:
Nitpick: Commented out code should be removed

Reply:
yes